source/admin_manual/submission_server.rst

```diff
@@ -59,19 +57,19 @@ Features
 The following SMTP capabilities are supported by the Dovecot submission
 service:
 
- * `8BITMIME <https://tools.ietf.org/html/rfc6152>`_ - Only if relay MTA provides
+ * 8BITMIME :rfc:`6152` - Only if relay MTA provides
    support
- * `AUTH <https://tools.ietf.org/html/rfc4954>`_
- * `BURL <https://tools.ietf.org/html/rfc4468>`_
- * `CHUNKING <https://tools.ietf.org/html/rfc3030>`_
- * `DSN <https://tools.ietf.org/html/rfc3461>`_ - Only if relay MTA provides
+ * AUTH :rfc:`4954`
+ * BURL :rfc:`4468`
+ * CHUNKING :rfc:`3030`
+ * DSN :rfc:`3461` - Only if relay MTA provides
    support
- * `ENHANCEDSTATUSCODES <https://tools.ietf.org/html/rfc2034>`_
- * `PIPELINING <https://tools.ietf.org/html/rfc2920>`_
- * `SIZE <https://tools.ietf.org/html/rfc1870>`_
- * `STARTTLS <https://tools.ietf.org/html/rfc3207>`_
- * `VRFY <https://tools.ietf.org/html/rfc5321>`_
- * `XCLIENT <http://www.postfix.org/XCLIENT_README.html>`_
+ * ENHANCEDSTATUSCODES :rfc:`2034`
+ * PIPELINING :rfc:`2920`
+ * SIZE :rfc:`1870`
+ * STARTTLS :rfc:`3207`
+ * VRFY :rfc:`5321`
+ * XCLIENT <http://www.postfix.org/XCLIENT_README.html>
```

Comment:
If I understand the patch correctly you want to display the link behind the feature. Then I would remove the `<` and `>` chars, that are put verbatim into the text now.

Reply:
was not really the intention, but i'll do something about it.